config/pizza-grant.yaml

```diff
@@ -7,12 +7,12 @@ channels:
     sla:
       responseTime: 1h
   - name: pizza-party-approval
+    id: C05RZATA3QR
     owners:
       default: [thomas, arpan]
```

Comment:
If the only item in the work queue is unassigned and matches w/ the india pattern, am I to understand that if thomas did a gimme, thomas would receive that work?

Reply:
okay - we might want to warn ppl when they are pulling work from a specialized/owned resource - that can be a separate feature though I guess...